terraform/aws/eks-ebs-csi-addon/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "us-west-2"
+}
+
+variable "vpc_name" {
+  type = string
+  default = "example-vpc"
+}
+
+variable "cluster_name" {
+  type = string
+  default = "example-cluster"
+}
+
+variable "cluster_version" {
+  type = string
+  default = "1.23"
+}
+
+variable "node_group_one_name" {
+  type = string
+  default = "ng-example-1"
+}
+
+#variable "node_group_two_name" {
+# type = string
+#  default = "ng-example-2"
+#}
+
+variable "instance_type_one" {
+  type = string
+  default = "t3.small"
```

Comment:
Should we use t3.medium instead to ensure customer can install Kubecost with best exp?

Reply:
Works for me! I will make the change.

Reply:
This change is up.